spec/features/admin_user/admin_order_index_spec.rb

```diff
@@ -0,0 +1,170 @@
+require "rails_helper"
+
+RSpec.feature "the admin can visit the orders page" do
+
+  let(:category)  {Category.create(name: "Category")}
+
+  let(:admin)  {User.create(full_name: "Admin",
+    email: "admin@admin.com",
+    password: "password",
+    role: 1)}
+  let(:user)   {User.create(full_name: "User",
+    email: "user@user.com",
+    password: "password",
+    role: 0)}
+
+  let(:item1)   {Item.create(title: "Item 1",
+    description: "I'm active!",
+    price: 9.99,
+    categories: [category])}
+  let(:item2)   {Item.create(title: "Item 2",
+    description: "I'm active!",
+    price: 9.99,
+    categories: [category])}
+  
+  let(:order1)  {Order.create(items: [item1],
+                              user_id: user.id,
+                              total_price: item1.price)}
+  let(:order2)  {Order.create(items: [item2],
+                              user_id: user.id,
+                              total_price: item2.price,
+                              status: "paid")}
+  let(:order3)  {Order.create(items: [item1, item2],
+                              user_id: user.id,
+                              total_price: item1.price + item2.price,
+                              status: "cancelled")}
+  let(:order4)  {Order.create(items: [item1, item2],
```

Comment:
Avoid using {...} for multi-line blocks.<br>Unnecessary spacing detected.<br>Block body expression is on the same line as the block start.